shimmer/develop/what-is-shimmer/layer-1-landscape.md

```diff
@@ -0,0 +1,141 @@
+| [Storage deposits](#storage-deposits)                          | None                                                                                                   | Yes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| [Output unlock conditions](#output-unlock-conditions)          | [Address](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/) | [Address](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/), [Storage Deposit Returns](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#storage-deposit-return), [Expiration](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#expiration), [Timelock](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#timelock) |
+| [IOTA Smart Contract (ISC)](#iota-smart-contracts-isc) Support | No                                                                                                     | On a [private Tangle](https://wiki.iota.org/shimmer/smart-contracts/guide/development_tools/docker_preconfigured/)                                                                                                                                                                                                                                                                                                                                                                            |
+
+## Stability
+
+Shimmer is designed as a staging network to test features publicly before deploying them on the IOTA network, so it
+will always have more features than IOTA. As this may come at the cost of some stability, before any protocol changes are deployed on the [Shimmer Mainnet](https://wiki.iota.org/shimmer/develop/endpoints/shimmer/), they have been tested
+in the [Shimmer Testnet](https://wiki.iota.org/shimmer/develop/endpoints/testnet/).
+
+## Store Data
+
+### IOTA
+
+In IOTA, you can attach
+an [IndexationPayload](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#indexationpayload)
+to
+any [SignedTransaction](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#signedtransaction).
```

Comment:
Indexation payload doesn't need to be part of a transaction. It can also be a standalone message.
```suggestion
In IOTA, you can attach
an [IndexationPayload](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#indexationpayload)
to a [message](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#messages) or
any [SignedTransaction](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#signedtransaction).
```